.travis.yml

```diff
@@ -7,8 +7,7 @@ notifications:
   email: false
 node_js:
   - 4
-before_script:
-  - npm prune
```

Comment:
can you leave the `npm prune` in? If we don’t have it in store-server, can you send a PR for it? As we cache the `node_modules` folder, this can save us lots of headaches